central/pom.xml

```diff
@@ -72,19 +72,19 @@
     <dependency>
       <groupId>org.infinispan</groupId>
       <artifactId>infinispan-core</artifactId>
-      <version>10.1.1.Final</version>
+      <version>9.4.16.Final</version>
```

Comment:
can you add a comment above this line explaining, so that I don't forget and bump the version again?